### PR TITLE
adding kwargs to trunk ports and adding stp_edge unit tests

### PR DESCRIPTION
**Pull request by radez**

### CODE REVIEW

File: tests/unit/test_api.py [hunk 1]
```diff
@@ -144,6 +151,15 @@ def test_assign_trunk_port(self, m_run_task):
...
         m_run_task.assert_called_once()
 
+    def test_assign_trunk_port_stp_edge(self, m_run_task):
+        self.net_runr.conf_trunk_port(self.testhost,
+                                      self.testport,
+                                      self.testvlan,
+                                      trunked_vlans=self.testvlans,
+                                      stp_edge=True)
+
+        m_run_task.assert_called_once()
```
Comment (michaeltchapman) [inline tests/unit/test_api.py]: I think this and the other should assert_called_once_with to make sure the vars are passed through

**Comment by tzumainn** [comment]
Hi! We've tested this to great success; any word on when this might be merged... ?

**Comment by radez** [comment]
We'll put gate on it and get it merged.

**Comment by tzumainn** [comment]
Great - thanks!

**Pull request merged**